### DISCUSSION TITLE
【求助】需要值得信赖地人认证QQ账号

### DISCUSSION THREAD

**Discussion by Dantehhh10** [share121/inter-knot#3437] [其他]
![images](https://github.com/user-attachments/assets/56302f92-cfb8-4149-a73e-bc8947600711)

俺系外地冷呐，想注册QQ账号，但系需要好友扫描二维码才嫩注册账号，有无有人愿意帮助俺扫描二维码通过认证呐

**Comment by Dantehhh10** [comment]
保证呐，请诸位要相信俺呐

**Comment by Dantehhh10** [comment]
俺真的需要帮助

**Comment by Fintbo** [comment]
宫廷玉液酒?

**Comment by lllliufeng** [comment]
我唔系地人点办？（doge
